### PR TITLE
Sending sum of deltas of database metrics for sql server

### PR DESCRIPTION
**Pull request by gurleen-12**
instead of running separate query.

### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Testing Guidelines

An overview on [testing](https://github.com/DataDog/dd-agent/blob/master/tests/README.md)
is available in our contribution guidelines.

### Versioning

- [ ] Bumped the check version in `manifest.json`
- [ ] Bumped the check version in `datadog_checks/{integration}/__init__.py`
- [ ] Updated `CHANGELOG.md`. Please use `Unreleased` as the date in the title
  for the new section.
- [ ] If PR impacts documentation, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new) 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: sqlserver/datadog_checks/sqlserver/sqlserver.py [hunk 1]
```diff
@@ -851,6 +854,11 @@ def fetch_metric(self, cursor, rows, columns, tags, cached_metrics_data):
 
                         cached_metrics_data[context] = (report_value, current_timestamp)
                         report_value_in_rate = (report_value - value)/(current_timestamp  - timestamp)
+                        if columns[index] in cumulative_rate_metrics:
```
Comment (AtreyeeMaiti) [inline sqlserver/datadog_checks/sqlserver/sqlserver.py:857]: won't this start generating sqlserver metrics as cumulative for all its children?
for eg. if sqlquery has iops and sqldatabase has iops, sqlserver iops will be summation of all of that?

Comment (gurleen-12) [reply sqlserver/datadog_checks/sqlserver/sqlserver.py:857]: yes totally, but we don't have rate metrics for query entity type and as Ive removed it for sql server too, so now only databases have rate metrics.

Comment (AtreyeeMaiti) [reply sqlserver/datadog_checks/sqlserver/sqlserver.py:857]: hmm, not the cleanest approach :) 
Let us put a TODO and create a tech debt ticket (@ahalam can help us with that for this repo).

Comment (gurleen-12) [reply sqlserver/datadog_checks/sqlserver/sqlserver.py:857]: agree, In future, we can derive this through YAML file.

**Pull request merged**